src/styles/_catalog.scss

```
@@ -14,6 +14,11 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms;
```

don't use all, add all props separately

```suggestion
    transition: transform 300ms, color 300ms;
```